internal/commands/commands.go

```diff
@@ -149,6 +149,13 @@ Realm app will be exported as well.`,
 				Help:        "Displays a list of data tables. Each data table displays the Name and ID of one (1) secret.",
 				Command:     &secrets.CommandList{},
 			},
+			cli.CommandDefinition{
+				Use:         "delete",
+				Display:     "secret delete",
```

Comment:
[nit] We should probably use `secrets delete` here to be consistent with pluralization